index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const _ = require('lodash');
+const os = require('os');
+// object representing a robots.txt that blocks everything:
+const disallowAll = {
+  '*': ['/']
+};
+
+const defaults = {
+  debug: true,
+  envs: {
+    production: disallowAll
```

Comment:
production should not be disallowAll.  We should have production allow all and then wildcard disallow all